app/components/NavigationBarWrapper.js

```diff
@@ -63,12 +63,21 @@ const BottomContainer = styled.SafeAreaView`
 const themeNavBarBorder = ({ theme }) =>
   theme.navBarBorder || Colors.NAV_BAR_VIOLET;
 
-const Header = styled.View`
-  background-color: ${themeNavBar};
-  border-bottom-color: ${themeNavBarBorder};
-  border-bottom-width: 1px;
-  flex-direction: row;
-`;
+// Add header padding on Android
+const Header = isPlatformiOS()
+  ? styled.View`
+      background-color: ${themeNavBar};
+      border-bottom-color: ${themeNavBarBorder};
+      border-bottom-width: 1px;
+      flex-direction: row;
+    `
+  : styled.View`
+      background-color: ${themeNavBar};
+      border-bottom-color: ${themeNavBarBorder};
+      border-bottom-width: 1px;
+      padding-top: 10px;
```

Comment:
With styled components you inline the function which takes props as the first arg:

`padding-top: ${props => isPlatformiOS() ? 0 : 10}`

Reply:
Also this needs to be 24px to match the previous spacing:

<img width="700" alt="Screen Shot 2020-04-26 at 9 24 37 AM" src="https://user-images.githubusercontent.com/702990/80313440-c998b700-879f-11ea-8f76-ec78fb59acef.png">


I suggest inserting a spacing View, rather than top padding for Android, otherwise it makes the title text center into the Title bar div too high:

<img width="316" alt="Screen Shot 2020-04-26 at 9 22 25 AM" src="https://user-images.githubusercontent.com/702990/80313381-732b7880-879f-11ea-8177-ed76d92c6820.png">


And when you do this fix it might push the settings icon down on the main screen.. cant quite tell.